### PR TITLE
stricter grid support detection

### PR DESCRIPTION
**Pull request by andrew-nowak**
Pr Body

### CODE REVIEW

File: packages/utilities/psammead-styles/README.md [hunk 1]
```diff
@@ -64,6 +65,25 @@ const SomeStyledComponent = css`
...
+
+We have defined custom feature detection statements (using CSS `@supports` syntax) to provide workarounds for browser-specific bugs.
+
+- gridDetection: We experienced issues using `@supports (display:grid)` from browsers with buggy CSS Grid implementations, and found that detecting on `@supports (grid-template-columns: fit-content(200px))` gave more consistent detection.
+
+```js
+import { gridDetection } from '@bbc/psammead-styles/detection';
+
+const someGridUsingComponent = css`
+  @supports (${gridDetection}) {
```
Comment (ghost) [inline packages/utilities/psammead-styles/README.md]:
```suggestion
  @supports (${grid}) {
```
reads a little bit nicer IMO.

**Review by ghost — CHANGES_REQUESTED** [review]
Update `package.json`, `package-lock.json` and `CHANGELOG.md`

File: packages/utilities/psammead-styles/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.2.0 | [PR#?](https://github.com/bbc/psammead/pull/?) Add custom CSS Grid feature detector |
```
Comment (ghost) [inline packages/utilities/psammead-styles/CHANGELOG.md]:
```suggestion
| 1.2.0 | [PR#1234](https://github.com/bbc/psammead/pull/1234) Add custom CSS Grid feature detector |
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by dr3 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by david-boydell** [comment]
Cool, thanks for the test page @aacn500, much appreciated. This is ready for merge.

**Pull request merged**